vllm/compilation/backends.py

```diff
@@ -261,14 +316,27 @@ class VllmBackend:
     split_gm: fx.GraphModule
     piecewise_graphs: List[SplitItem]
     returned_callable: Callable
+    # Inductor passes to run on the graph pre-defunctionalization
+    post_grad_passes: Sequence[Callable]
 
-    def __init__(self, ):
+    def __init__(self, post_grad_passes: Sequence[Callable]):
         # every instance of VllmBackend has its own graph pool
         self.graph_pool = torch.cuda.graph_pool_handle()
+        self.post_grad_passes = post_grad_passes
 
         # `torch.compile` is JIT compiled, so we don't need to
         # do anything here
 
+    def add_passes_to_config(self):
+        config = self.compilation_configs.inductor_compile_config
```

Comment:
why do we move it from `inductor_passes` to `inductor_compile_config`?

I separate them out, because we need to convert function qualname to function object in passes, while the rest configs are taken as-is.

Reply:
We don't, I left that part unchanged. This is just additional built-in passes that go to `ppst_grad_custom_post_pass`.